app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+require 'sidekiq/form526_job_status_tracker/job_tracker'
+require 'sidekiq/form526_job_status_tracker/metrics'
+
+module RapidReadyForDecision
+  class Form526BaseJob
```

Comment:
Running `diff app/workers/rapid_ready_for_decision/form526_base_job.rb app/workers/rapid_ready_for_decision/disability_compensation_job.rb` shows that most of the code in this new `Form526BaseJob` class is copied from `DisabilityCompensationJob`.